### DISCUSSION TITLE
Why is the search layout ignored on my website?

### DISCUSSION THREAD

**Discussion by BsNoSi** [datenstrom/community#487] [Ask a question] [answered]
I am using Datenstrom Yellow 0.8.10 with just updated search. Changes in "Search.html" are ignored – I even can remove it (!) without error message.

Search results are nevertheless generated but – and this is the reason for investigating this – with missing `</DIV>` entries that are part of (original) "search.html" but not generated in output.

This corrupts my layout.

I have to mention that I copied for testing reasons the original file into another layout file and renamed the used layout in `content/search/page.md`. The result is `[searchResultsUnlisted]`.

**Comment by markseuffert** [comment, marked as answer]
I would use `Layout: search` and then customise in file `system/layouts/search.html`.

**Comment by BsNoSi** [comment]
This is what I am trying for hours now. I will check this with another site, possibly it is a side effect of this specific one. But currently I don't have any idea which reason can cause this.

**Comment by markseuffert** [comment]
In case you are stuck, you can do trouble shooting inside the core and see **exactly** what is going on. Open file `system/extensions/core.php`, change the first line to `<?php define("DEBUG", 2);` and now look for something like `YellowPage::includeLayout`. Hope this helps.
